--- a/vllm/model_executor/models/registry.py
+++ b/vllm/model_executor/models/registry.py
@@ -341,6 +341,10 @@
 
 _MULTIMODAL_MODELS = {
     # [Decoder-only]
+    "Apertus1p5ForConditionalGeneration": (
+        "apertus_mm",
+        "Apertus1p5ForConditionalGeneration",
+    ),
     "AriaForConditionalGeneration": ("aria", "AriaForConditionalGeneration"),
     "AudioFlamingo3ForConditionalGeneration": (
         "audioflamingo3",
